sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInfo.scala

```diff
@@ -9,8 +9,8 @@ import data.{Ref, Relation}
 
 final class PackageInfo private[daml] (
     val mainPackageId: Ref.PackageId,
-    val mainPackageMetadata: Ast.PackageMetadata,
-    val pkgSignature: Map[Ref.PackageId, Ast.GenPackage[_]],
+    val mainPackage: Ast.Package,
+    val pkgSignature: Map[Ref.PackageId, Ast.Package],
```

Comment:
by convention a packageSignature is either `Map[Ref.PackageId, Ast.PackageSignature]` or `Map[Ref.PackageId, Ast.GenPackage[_]]`]

Reply:
Reverted as per our discussion offline